### ISSUE TITLE
Optimize: bug when not expanding Protect

### ISSUE THREAD

**Issue by Sciss** [Sciss/Negatum#8]
In GitLab by @Sciss on Jun 28, 2020, 13:22

input:
```scala
    NegatumIn()
    val lFDClipNoise    = LFDClipNoise.ar(0.025)
    val min             = lFDClipNoise min 0.64
    val median          = Median.ar(2211.0, length = 31.0)
    val freq_0          = Protect(median, -inf, inf, false) // this is causing trouble
    val lFDNoise3       = LFDNoise3.ar(freq_0)
    val lFDNoise1       = LFDNoise1.ar(2538.0)
    val lFPulse         = LFPulse.ar(freq = 337.99133, iphase = 0.0, width = 1.0)
    val mix             = Mix(
      Seq[GE](min, lFDNoise3, 1.0, lFDNoise1, lFPulse))
    NegatumOut(mix)
```
output:
```scala
    NegatumIn()
    val lFDNoise1 = LFDNoise1.ar(2538.0)
    val lFDNoise3 = LFDNoise3.ar(2211.0)
    val lFPulse   = LFPulse.ar(freq = 337.99133, iphase = 0.0, width = 1.0)
    val dC        = DC.ar(2242.64)
    val mix       = Mix(Seq[GE](lFDNoise1, lFDNoise3, lFPulse, dC))
    NegatumOut(mix)
```
We get a crazy DC which shouldn't happen.

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 13:22

changed the description

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 18:51

mentioned in commit 963d62230cfac692a33d29fb28bdf1c139f1e63e

**Comment by Sciss** [comment]
In GitLab by @Sciss on Jun 28, 2020, 22:31

closed via commit 963d62230cfac692a33d29fb28bdf1c139f1e63e
